README.md

```diff
@@ -42,3 +42,16 @@ python -m unittest
 This set of tests is configured using the `integration_configuration.yaml` file, which contains information for which
 units are to be present in a test, and the order they're to be run in. It also contains general settings, such as
 where the test fixtures are located, and which files need to be cleaned up when a test job is complete.
+
+## Application Features
```

Comment:
The termin "feature" is not well defined. What exactly is it?

I suggest we just use `model` instead. Have a `models` folder and application subfolders inside it accordingly